coriolis/providers/backup_writers.py

```diff
@@ -300,7 +327,7 @@ def _connect_ssh(self):
                 username=self._username,
                 pkey=self._pkey,
                 password=self._password)
-        except:
+        except BaseException:
```

Comment:
NIT: I would personally go for an explicit `except (Exception, KeyboardInterrupt)`, as `KeyboardInterrupt` is the only other "level 2 exception" we're actually expecting.

Reply:
done